### ISSUE TITLE
Missing error message in debug

### ISSUE THREAD

**Issue by ktisha** [sproshev/TheRPlugin#20]
File "tmp.R":
```R
Square()
```

On executing tmp.R I got:
```
/usr/bin/R --slave -f /home/ktisha/PycharmProjects/TheRProject/tmp1.r
Error: could not find function "Square"
Execution halted

Process finished with exit code 1
```

No error message in debugger.

**Comment by sproshev** [comment]
The problem is in TheRProcessReceiver which doesn't check error stream of interpreter process.

**Comment by sproshev** [comment]
Fixed in [e416af900242618ee724fc3f7d8b4c00b844b73d, .., 407c7740e2549535d26fd62e27eda05432c3a6b8]
